createNPMPackage.sh

```diff
@@ -3,62 +3,27 @@ set -e
 
 ROOT=$(pwd)
 
-# part I (prepare android-npm)
-
-cd Example/android/
-./gradlew assembleRelease
-cd ../node_modules/react-native/ReactAndroid/build/react-ndk/all
-
-for d in *
-do
-  echo "processing $d"
-  mkdir -p  "${ROOT}/android/src/main/JNILibs/${d}"
-  cp "${d}/libreanimated.so" "${ROOT}/android/src/main/JNILibs/${d}/."
-done
-
-cd "${ROOT}"
-rm -r android/build/outputs/aar/*.aar
-
-cd  Example/android/
+# part I - I (clear old aar)
 
+rm -rf android/build/outputs/aar/*.aar
 
- ./gradlew clean
- ./gradlew aR
-
-# clean so
-
-
-cd ../node_modules/react-native/ReactAndroid/build/react-ndk/all
-
-rm -rf "${ROOT}/android/src/main/JNILibs/"
+# part I (prepare android-npm)
 
+cd Example/android
+./gradlew :react-native-reanimated:assembleDebug
 cd $ROOT
 
-rm -f android-npm/react-native-reanimated.aar
+rm -rf android-npm/react-native-reanimated.aar
 cp android/build/outputs/aar/*.aar android-npm/react-native-reanimated.aar
 
-# PART II (create package)
-
-
 mv android android-temp
 mv android-npm android
 
-npm pack
+# PART II (create package)
 
-#mv android android-npm
-#mv android-temp android
+npm pack
 
+mv android android-npm
```

Comment:
I'd prefer to avoid having that `android-npm` folder and doing this move here. Afraid of being hit one time with a problem when build fails in the middle and the folder isn't restored.

Reply:
I don't see what could go wrong here. 
Commands are executed synchronously in our script.
If you see a better solution then I will change it.

Reply:
If you want we can create another directory with android-npm (renamed to android) there.  With symlinks to package.json, plugin, iOS, and other dirs/files used during publish/pack.